components/brave_wallet/browser/brave_wallet_service.h

```diff
@@ -162,6 +162,10 @@ class BraveWalletService : public KeyedService,
       const std::string& key,
       IsBase58EncodedSolanaPubkeyCallback callback) override;
 
+  void Base58EncodeAddresses(
+      const std::vector<std::vector<std::uint8_t>>& addresses,
+      Base58EncodeAddressesCallback callback) override;
+
```

Comment:
We could also just name this Base58Encode since it's not limited to just addresses.

Reply:
yea we can.